charts/azuremonitor-containers/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 7.0.0-1
 description: Helm chart for deploying Azure Monitor container monitoring agent in Kubernetes
 name: azuremonitor-containers
-version: 2.7.4
+version: 2.7.6
```

Comment:
Why is this updated by 2 minor versions?

Reply:
> Why is this updated by 2 minor versions?

> Because for non-AKS, we had two releases. one is with image ciprod09162020 and other one is ciprod09252020 with mdm hotfix i.e. azure json crash. This is version in sync with what we have here https://github.com/helm/charts/commit/ea2a5fa08db309b80ac658af98e0c2195e2a8acc#diff-22f9aff5b169d55c0e800ac6b861272a.